code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -214,6 +215,24 @@ proc/issyndicate(mob/living/M as mob)
 	syndobj.owner = syndicate
 	syndicate.objectives += syndobj
 
+/datum/game_mode/proc/forge_sleeper_objectives(datum/mind/sleeper_agent, custom_objective)
+	var/datum/objective/nuclear_helper/syndobj = new
+	if(custom_objective == "")	//if people are too lazy to give a customised objective
+		custom_objective = "Follow all orders given by Nuclear Operatives."
+	var/datum/objective/custom_obj = new(custom_objective)
+	custom_obj.owner = sleeper_agent
+	custom_obj.completed = TRUE	//Defaults to completed, admins can toggle this if the sleeper fucks up
+	syndobj.owner = sleeper_agent
+	sleeper_agent.objectives += syndobj
+	sleeper_agent.objectives += custom_obj
+
+/datum/game_mode/proc/greet_sleeper(datum/mind/sleeper_agent)
+	SEND_SOUND(sleeper_agent.current, 'sound/ambience/antag/tatoralert.ogg')
+	to_chat(sleeper_agent.current, "<span class='notice'>You suddenly realize you are a Syndicate sleeper agent that has been activated in a rush. Nuclear operatives are on their way to destroy the station. Aid them in this task and fulfil your objective. <b> Be aware that they outrank you, their orders are to be followed</b></span>")

```

Comment:
I would use the same wording as when you're auto traitored, with the extra info about helping nuke ops etc added below.
"<span class='danger'>ATTENTION:</span> It is time to pay your debt to the Syndicate..."
"<B>You are now a traitor.</B>"

Reply:
Hmm, true. Two messages work, too. Will get to that, later.